src/Groups/GAPGroups.jl

```diff
@@ -349,8 +349,7 @@ Base.:/(x::GAPGroupElem, y::GAPGroupElem) = group_element(parent(x), (x.X / y.X)
 
 Base.:\(x::GAPGroupElem, y::GAPGroupElem) = group_element(parent(x), (x.X \ y.X)::GapObj)
 
-# Compatibility with GroupsCore interface
-one!(x::GAPGroupElem) = one(parent(x))
+# Compatibility with groups interface
 inv!(out::GAPGroupElem, x::GAPGroupElem) = inv(x)  #if needed later
```

Comment:
```suggestion
```
All of the following should go if they are equal to the default implementation in AA